src/main/kotlin/com/openlattice/util/Hashcodes.kt

```diff
@@ -0,0 +1,53 @@
+package com.openlattice.util
+
+/**
+ * @author Drew Bailey &lt;drew@openlattice.com&gt;
+ */
+class Hashcodes private constructor(){
```

Comment:
Could we just filter nulls and invoke https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#hash-java.lang.Object...- for all of these?